e2e/nomostest/new.go

```diff
@@ -252,9 +253,11 @@ func FreshTestEnv(t nomostesting.NTB, opts *ntopts.New) *NT {
 	// Speed up the delay between sync attempts to speed up testing
 	// (default: 15s), but leave the remediator (paused while syncing) and retry
 	// code (1s delay) time to execute between sync attempts.
-	nt.ReconcilerPollingPeriod = 2 * time.Second
+	nt.ReconcilerPollingPeriod = 15 * time.Second
 	// Speed up the delay between rendering attempts to speed up testing (default: 5s)
-	nt.HydrationPollingPeriod = 1 * time.Second
+	nt.HydrationPollingPeriod = 5 * time.Second
```

Comment:
Had to bump these back to the default to address test failures, see https://github.com/GoogleContainerTools/kpt-config-sync/pull/707#discussion_r1241011085